Sources/Private/NodeRenderSystem/Nodes/RenderNodes/StrokeNode.swift

```diff
@@ -136,17 +136,16 @@ extension Array where Element == DashElement {
   /// representations usable in a `CAShapeLayer`
   var shapeLayerConfiguration: (
     dashPatterns: ContiguousArray<ContiguousArray<Keyframe<Vector1D>>>,
-    dashPhase: ContiguousArray<Keyframe<Vector1D>>)
-  {
-    var dashPatterns = ContiguousArray<ContiguousArray<Keyframe<Vector1D>>>()
-    var dashPhase = ContiguousArray<Keyframe<Vector1D>>()
-    for dash in self {
-      if dash.type == .offset {
-        dashPhase = dash.value.keyframes
-      } else {
-        dashPatterns.append(dash.value.keyframes)
+    dashPhase: ContiguousArray<Keyframe<Vector1D>>) {
+      var dashPatterns = ContiguousArray<ContiguousArray<Keyframe<Vector1D>>>()
```

Comment:
This is tough to read, could we maybe make a typealias for this type?

Reply:
sure

Reply:
I wonder why SwiftFormat indented this property, must be an obscure implementation bug that I haven't hit before